continuous-integration/requirements.Rmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Requirements for continuous integration"
+maintainer: "Thierry Onkelinx"
+output: html_document
+---
+
+# Must have
+
+- start automatically when pushing a git repository on Github or Bitbucket
+- start automatically when creating a pull request on Github or Bitbucket
```

Comment:
Suppose that someone forks a repo, applies some changes but doesn't test them. Then they seed a PR to our repository. At that point the CI should fire up and test the PR. We can setup Github to block merging the PR until the CI give the all clear. This avoid introducing code into the repo with known problems.